### DISCUSSION TITLE
Is it possible to have multiple processors for the same file extension?

### DISCUSSION THREAD

**Discussion by grenganathan18** [eslint/eslint#16951] [Q&A] [answered]
My project uses `eslint-plugin-json` and it internally uses a custom processor for `.json` files. Now I want to add a new custom rule for `.json` files and `eslint-plugin-json` does not provide that. Is it possible to configure eslint to run both the json processor  in `eslint-plugin-json` and a custom processor that I will be writing for the same '.json' file extension?

Did a POC and it looks like only one of the processor runs. Any ideas?

**Comment by nzakas** [comment, marked as answer]
You can only use one processor:
https://eslint.org/docs/latest/use/configure/plugins#specify-a-processor
